core/src/main/java/org/mobilitydata/gtfsvalidator/table/GtfsFeedLoader.java

```diff
@@ -100,6 +100,9 @@ public GtfsFeedContainer loadAndValidate(
     Map<String, GtfsTableDescriptor<?>> remainingDescriptors =
         (Map<String, GtfsTableDescriptor<?>>) tableDescriptors.clone();
     for (String filename : gtfsInput.getFilenames()) {
+      if (filename.contains("translations")) {
```

Comment:
If I am not mistaken, the code depends on the order of the filenames obtained with getFilenames, i.e. it requires that translations is processed before feed_into. I don't think this is guaranteed
It would be better to have one loop that checks for the presence of translations, then the original loop that processes the files.